scripts/model/blog.js

```diff
@@ -21,7 +30,10 @@
     Post.allArticles = inputdata.sort(function(firstEle, secondEle){
       return (new Date(secondEle.publishedOn)) - (new Date(firstEle.publishedOn));
     }).map(function(ele){
-      return new Post(ele);
+      var post = new Post(ele);
+      post.daysAgo = parseInt((new Date() - new Date(post.publishedOn))/60/60/24/1000);
+      post.publishStatus = post.publishedOn ? 'published ' + post.daysAgo + ' days ago' : '(draft)';
+      return post;
```

Comment:
Really like the code above. Seems to work well. Nice job. Great use of chaining.